zyngine/zynthian_engine_pianoteq.py

```diff
@@ -211,12 +212,18 @@ def fix_pianoteq_config():
 
 PIANOTEQ_CONFIG_FILE =  PIANOTEQ_CONFIG_DIR + "/" + PIANOTEQ_CONFIG_FILENAME
 
+jack_client = jack.Client("zynpianoteq")
```

Comment:
I just added 2 functions to the main GUI class:

+ get_jackd_samplerate()
+ get_jackd_blocksize()

I think it's better to have this info easily available for all engines (zynaddsubfx, etc.)

Please, modify the code for using the new functions like:

    PIANOTEQ_SAMPLERATE = self.zyngui.get_jackd_samplerate()

Reply:
OK! I've already changed the code.